addons/html_builder/static/src/plugins/gallery_element_option.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+<t t-name="html_builder.GalleryElementOption">
+    <BuilderRow label.translate="Re-order">
+        <BuilderButtonGroup>
+            <BuilderButton icon="'fa-angle-double-left'" preview="false" title.translate="Move to first" action="'setGalleryElementPosition'" actionValue="'first'"/>
+            <BuilderButton icon="'fa-angle-left'" preview="false" title.translate="Move to previous" action="'setGalleryElementPosition'" actionValue="'prev'"/>
+            <BuilderButton icon="'fa-angle-right'" preview="false" title.translate="Move to next" action="'setGalleryElementPosition'" actionValue="'next'"/>
+            <BuilderButton icon="'fa-angle-double-right'" preview="false" title.translate="Move to last" action="'setGalleryElementPosition'" actionValue="'last'"/>
+        </BuilderButtonGroup>
+    </BuilderRow>
```

Comment:
I think that you can lift up redondant props in children to the parent. The parent will then pass it down trough the context thanks to `useBuilderComponent`
```suggestion
        <BuilderButtonGroup preview="false" action="'setGalleryElementPosition'">
            <BuilderButton icon="'fa-angle-double-left'" title.translate="Move to first" actionValue="'first'"/>
            <BuilderButton icon="'fa-angle-left'" title.translate="Move to previous" actionValue="'prev'"/>
            <BuilderButton icon="'fa-angle-right'" title.translate="Move to next" actionValue="'next'"/>
            <BuilderButton icon="'fa-angle-double-right'" title.translate="Move to last"  actionValue="'last'"/>
        </BuilderButtonGroup>
    </BuilderRow>
```